Gemfile

```diff
@@ -24,6 +24,8 @@ gem 'rails-controller-testing', group: :test
 
 gemspec
 
+gem 'solidus_support', github: 'solidusio/solidus_support'
```

Comment:
I've just released `solidus_support`0.5.0 with `SolidusSupport::EngineExtensions`.  We should be able to remove this line now.

Reply:
Hey @aldesantis, when you have some time, can you please remove this line so we'll be able to merge this?